reaper/http.go

```diff
@@ -154,6 +158,14 @@ func (c *client) addCommonHeaders(req *http.Request) {
 	if c.userAgent != "" {
 		req.Header.Set("User-Agent", c.userAgent)
 	}
+
+	if c.jSessionId != nil {
```

Comment:
Just a detail. Since we have distinct methods for common headers and form data headers, how about we isolate the auth headers in a separate method?

Reply:
Sounds good 👍